tests/e2e/setup.go

```diff
@@ -162,7 +162,7 @@ func (s *TestSuite) QueryBaseFee() sdk.DecCoins {
 	// create the oracle client
 	c := types.NewQueryClient(cc)
 
-	resp, err := c.BaseFee(context.Background(), &types.BaseFeeRequest{})
+	resp, err := c.BaseFee(context.Background(), &types.GasPriceRequest{})
```

Comment:
This query needs an argument no?

e2e function should be renamed from `BaseFee`